mmv1/products/healthcare/terraform.yaml

```diff
@@ -82,6 +82,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           dataset_name: "example-dataset"
           dicom_store_name: "example-dicom-store"
           pubsub_topic: "dicom-notifications"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_dicom_store_bq_stream"
+        primary_resource_id: "default"
+        vars:
+          dataset_name: "example-dataset"
```

Comment:
if any of these resources are subject to quotas, it would be great to prefix their names in tests with `tf-test-` - you can do this with the `test_vars_overrides` option.

Reply:
https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-4717-old..auto-pr-4717#diff-c24c62b3da0449f52cc13e6ebc7b57bfb0a2f511d98a0514e27326c6c94c3415R129 I saw it gen `tf-test` prefix, but I don't know why.

Reply:
you're right - it does that automatically 🎉 my bad.